pmm/pmm2-portal-integration.groovy

```diff
@@ -0,0 +1,149 @@
+library changelog: false, identifier: 'lib@master', retriever: modernSCM([
+    $class: 'GitSCMSource',
+    remote: 'https://github.com/Percona-Lab/jenkins-pipelines.git'
+]) _
+
+def latestVersion = pmmVersion()
+def versionsList = pmmVersion('list_with_old')
+def getMinorVersion(VERSION) {
+    return VERSION.split("\\.")[1].toInteger()
+}
+
+pipeline {
+    agent {
+        label 'docker-farm'
+    }
+    environment {
+        PORTAL_USER_EMAIL=credentials('PORTAL_USER_EMAIL')
+        PORTAL_USER_PASSWORD=credentials('PORTAL_USER_PASSWORD')
+        OKTA_TOKEN=credentials('OKTA_TOKEN')
+        SERVICENOW_LOGIN=credentials('SERVICENOW_LOGIN')
+        SERVICENOW_PASSWORD=credentials('SERVICENOW_PASSWORD')
+        SERVICENOW_DEV_URL=credentials('SERVICENOW_DEV_URL')
+        OAUTH_DEV_CLIENT_ID=credentials('OAUTH_DEV_CLIENT_ID')
+        PORTAL_BASE_URL=credentials('PORTAL_BASE_URL')
+    }
+    parameters {
+        string(
+            defaultValue: 'main',
+            description: 'Tag/Branch for UI Tests Repo integration with Portal',
+            name: 'GIT_BRANCH')
+        string(
+            defaultValue: 'perconalab/pmm-server:dev-latest',
+            description: 'PMM Server docker tag to test for integration with Portal',
+            name: 'DOCKER_TAG')
+        choice(
+            choices: versionsList,
+            description: 'PMM Server Version to test for integration with Portal',
+            name: 'DOCKER_VERSION')
+        string(
+            defaultValue: latestVersion,
+            description: 'latest PMM Server Version',
+            name: 'PMM_SERVER_LATEST')
+        string(
+            defaultValue: 'main',
+            description: 'Tag/Branch for pmm-qa repository',
+            name: 'PMM_QA_GIT_BRANCH')
+    }
+    options {
```

Comment:
what or how we will trigger this job? there is no autobuild trigger for this one?